build/.goreleaser.yml

```diff
@@ -7,16 +7,16 @@ builds:
   goos:
    - darwin
    - linux
+   - windows
   goarch:
    - amd64
   env:
    - CGO_ENABLED=0
 archive:
   files:
     - "/workspace/_output/{{ .Os }}_{{ .Arch }}/kubebuilder/bin/*"
-  wrap_in_directory: true
 # wrap all the binaries under 'bin/' dir
-  wrap_in_directory_path: "bin"
+  wrap_in_directory: "bin"
```

Comment:
When I installed the latest version of goreleaser, using `wrap_in_directory_path` threw an error for me. Can revert if this is a versioning issue, but I had to use this locally.

Reply:
I don't remember the exact change, but we had to modify goreleaser to add this additional field to support packaging multiple binaries under a "bin" directory. So this change will break the build steps in cloud-build, so lets revert this.

Reply:
@droot reverted